start-site/src/main/java/io/spring/start/site/extension/springcloud/SpringCloudProjectsMetadataProvider.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2012-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.spring.start.site.extension.springcloud;
+
+/**
+ * Provides the {@link SpringCloudProjectsMetadata} for a given release train version.
+ *
+ * @author Olga Maciaszek-Sharma
+ */
+public interface SpringCloudProjectsMetadataProvider {
+
+	/**
+	 * Returns Spring Cloud projects metadata for the specified Spring Cloud Release Train
+	 * version.
+	 * @param releaseTrainVersion release Train version as {@link String}
+	 * @return {@link SpringCloudProjectsMetadata} object
+	 */
+	SpringCloudProjectsMetadata get(String releaseTrainVersion);
```

Comment:
That contract is very specific to Spring Cloud and I don't think that's warranted. Rather I would have expected something that would give us a map of version for a given `platformVersion`. 

For instance, I am calling a service for Spring Boot `2.1.5.RELEASE` and I get a bunch of versions, letting the service discover based on what it can discover. We'd get a `spring-cloud-contract` but also a `hibernate` key for the Hibernate version read from the Spring Boot bom.